headless-cms/head/components/link.tsx

```diff
@@ -7,25 +7,27 @@ type CustomLinkProps = {
 
 export default function CustomLink(props: CustomLinkProps) {
 	const { label, url } = props
+	const articlePrefix = '/blog'
 
 	const pageUrlRole: any = {
 		homePage: `/`,
-		blogPage: `/blog`,
+		blogPage: articlePrefix,
 		error404Page: "/404"
 	}
 
 	const urlTypes: any = {
 		url: url?.url,
 		page: url?.page?.role ? pageUrlRole[url?.page.role] : `/${url?.page?.slug}`,
-		article: `/${url?.article?.slug}`
+		article: `${articlePrefix}/${url?.article?.slug}`
 	}
 	const href = url ? url.type && urlTypes[url.type] : null
 
 	if (href) {
+		if (url.type === 'url') {
```

Comment:
This is a difference to the other one - why?

Reply:
Good catch. It should be same.